scripts/installation/install_solc.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
```

Comment:
Perhaps a comment in here about this script - It does *not* need nucypher installed to operate, but *does* depend on requests and is cross-platform compatible, etc.

Reply:
✔️